core/jobs/batch_jobs/exp_validation_jobs.py

```diff
@@ -0,0 +1,1002 @@
+                    if len(alt_with_value) < 5:
+                        rte_components_errors.append(
+                            'State - {key} Image tag alt value '.format(
+                                key=key) +
+                            'is less than 5.')
+
+                    # Validates if filepath extension is not svg.
+                    if file_with_value[-3:] != 'svg':
+                        rte_components_errors.append(
+                            'State - {key} Image tag filepath value '.format(
+                                key=key) +
+                            'does not have svg extension.')
+
+                # Validates if math tag filepath extension is not svg.
+                elif rte_component['id'] == 'oppia-noninteractive-math':
+                    svg_filename = rte_component['customization_args'][
+                        'math_content-with-value']['svg_filename']
+
+                    if svg_filename[-3:] != 'svg':
+                        rte_components_errors.append(
+                            'State - {key} Math tag svg_filename '.format(
+                                key=key) +
+                            'value has a non svg extension.')
+
+                # Validates if skillreview text value is empty.
+                elif (rte_component['id'] ==
+                    'oppia-noninteractive-skillreview'):
+                    text_with_value = rte_component['customization_args'][
+                        'text-with-value']
+
+                    if text_with_value == '':
+                        rte_components_errors.append(
+                            'State - {key} Skill review tag text '.format(
+                                key=key) +
+                            'value is empty.')
+
+                # Validates if video start value is greater than end value.
+                elif rte_component['id'] == 'oppia-noninteractive-video':
+                    start_value = (
+                        rte_component['customization_args']
+                        ['start-with-value'])
+                    end_value = (
+                        rte_component['customization_args']
+                        ['end-with-value'])
+
+                    if int(start_value) > int(end_value):
+                        rte_components_errors.append(
+                            'State - {key} Video tag start '.format(
+                                key=key) +
+                            'value is greater than end value.')
+
+                # Validates link url is not https and text value is not empty.
+                elif rte_component['id'] == 'oppia-noninteractive-link':
+                    url_value = rte_component['customization_args'][
+                        'url-with-value'][:5]
+                    text_with_value = rte_component['customization_args'][
+                        'text-with-value']
+
+                    if text_with_value == '':
```

Comment:
Same, I think we can only set the value to a string or leave it empty.
Thanks

Reply:
Are you 100% sure or not? If not better add a check for `None` too.

Reply:
Yes because this will be a `html` data present in the content variable and also there is no way to produce the test scenario where I can assign the value as `None`.
I can totally add the check for the None part but the backend tests will fail for not covering this particular part in the test file and we cannot assign the value `None`.

Reply:
I think you can add the None check in same if to "hack" the tests a bit.

Reply:
Done